src/prisma/schema.prisma

```diff
@@ -1994,3 +1995,13 @@ model FormFieldAnswer {
 
   @@index([fieldId])
 }
+
+model UserOTP {
+  id           String  @id @default(uuid()) @db.Uuid
+  secret       String
+  recoveryCode String?
```

Comment:
make this a separate table including createdAt and usedAt?

Reply:
add createdAt field on here, activatedAt field

Reply:
Done

Reply:
Added deletedAt